.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
huh dependabot looks kinda sick. will it remind us abt updating mbedos? i dont think we want that bc we dont update our mbed

Reply:
it will submit a pr that updates mbed os. we don't necessarily have to merge it but i think most of the time we should. reason being that its usually small marginal updates that won't break anything on our side when we update incrementally. but if we wait too long to do updates, we get a big mess like what i had to do a couple of months ago.